content/webapp/pages/guides/exhibitions/[id]/[type]/index.tsx

```diff
@@ -219,7 +173,7 @@ export const getServerSideProps: GetServerSideProps<
           serverData,
           type,
           userPreferenceSet,
-          stopId: stopId as string | undefined,
+          stopId: toMaybeString(stopId),
```

Comment:
This function returns `string | undefined`, where `stopId` has the potential to be `string[]`, so I thought it was a relevant change.